docker/solr/solrconfig.xml

```diff
@@ -1170,76 +1170,6 @@
 
     -->
 
-  <!-- Add unknown fields to the schema
```

Comment:
Is this the same code that allows people to add fields with the `_s`, `_ss`, `_t` type of syntax?  Because I think that might be really useful for some people who are looking to extend openoni.  At least, at UNL we've used it quite a lot in other projects based on solr building off of a default schema, though not specifically for chronam.